oci-unit-tests/standalone-checks_test.sh

```diff
@@ -181,17 +181,13 @@ test_images_hashes_are_equal()
                 cnt_images_list_tmp=$(echo "${images_list_tmp}" | wc -w)
 
                 if [ "${cnt_images_list}" -lt "${cnt_images_list_tmp}" ]; then
-                    echo "E: Registry '${registry}' contains more images than '${orig_registry}'.  Here are the extra images:" > /dev/stderr
+                    echo "W: Registry '${registry}' contains more images than '${orig_registry}'.  Here are the extra images:" > /dev/stderr
```

Comment:
Out of curiosity: do you have a reason to prefer `> /dev/stderr` over the more common `>&2`? My guess it: `> /dev/stderr` is more explicit and we don't care about portability to old/obscure systems, but I wonder if there's more :)

Reply:
No particular reason :-).  I'm just used to write code like this, so I just don't think about other ways nowadays ;-).